server/ollamacfg/config.go

```diff
@@ -0,0 +1,167 @@
+package ollamacfg
```

Comment:
nit: `config` might be cleaner name

Reply:
As a symbol name `config` is more generic and will conflict with more things.  What about `ocfg`?